gems/sorbet/lib/gem-generator-tracepoint/tracer.rb

```diff
@@ -147,8 +152,12 @@ def self.install_tracepoints
             begin
               tp.disable
 
-              singleton = tp.method_id == :singleton_method_added
+              singleton = method_id_sym == :singleton_method_added
               receiver = singleton ? Sorbet::Private::RealStdlib.real_singleton_class(tp.self) : tp.self
+
+              # JRuby the main Object is not a module
```

Comment:
can you say why you don't want to do this whole block of code in JRuby?

Reply:
It crashes -- the below code assumes its a module.

Reply:
`@real_instance_methods.bind(mod).call(arg)`

Reply:
I meant to say so in the code, sorry

Reply:
did it with Github a suggestion :)